virttest/utils_test/qemu/__init__.py

```diff
@@ -186,6 +186,32 @@ def clear_win_driver_verifier(driver, vm, timeout=300):
         session.close()
 
 
+@error_context.context_aware
+def check_win_driver(test, driver, vm, timeout=300):
+    """
+    Check if driver is installed and state is running for windows guest.
+    :param test: QEMU test object.
+    :param driver: The driver which needs enable the verifier.
+    :param vm: VM object.
+    :param timeout: Timeout in seconds.
```

Comment:
Better have a return doc string as well, as it is not well clear at first look,

Reply:
Agree, updated.